tests/genai/scorers/test_builtin_scorers.py

```diff
@@ -1820,6 +1823,173 @@ def test_conversational_role_adherence_instructions():
     assert "persona" in instructions.lower() or "boundaries" in instructions.lower()
 
 
+def test_knowledge_retention_uses_default_last_turn_scorer():
+    session_id = "test_session_default_scorer"
+    session = []
+
+    with mlflow.start_span(name="turn_0") as span:
```

Comment:
nit: might be worth moving some of this into a helper since it's repeated in multiple tests